packages/logs/src/boot/logsPublicApi.ts

```diff
@@ -87,6 +90,16 @@ export function makeLogsPublicApi(startLogsImpl: StartLogs) {
         return
       }
 
+      if (initConfiguration.storeContextsAcrossPages) {
+        // Note: context API calls before init are dismissed
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Why do we dismiss earlier API calls? Could you add a bit more context?

Reply:
The issue is that before init, we provide a fully functioning context manager and the intent of using the storage is only provided at init.

IMO, the right way to solve that would be to buffer context manager API calls and only applying them at init but that would be a breaking change.

In the mean time, there is a priority issue between the context in memory before init and the context in local storage if we want to reconcile them, which one should overwrite the other?
So, the current approach do not reconcile them.

wdyt?

Reply:
I don't see the point of buffering context manager API calls? We could have a boolean flag to store whether the context did change before init, and if it has, store the current context in local storage. Or am I missing something?

Reply:
Same as after the init call, `{ foo: 'qux' }`?

Reply:
Discussed IRL, let's combine contexts for now